content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,385 @@
+            - AcmeInc
+```
+
+The owner picker, allows the user to select a user / group in the Backstage catalog. e.g.
+```yaml
+  parameters:
+    properties:
+      owner:
+        type: string
+        ui:field: OwnerPicker
+```
+
+## `number`
+You can allow the user to enter a number using the `number` type:
+
+```yaml
+  parameters:
+    properties:
+      size:
+        type: number
+```
+
+## `object`
+
+The `object` allows the collection of more complex types of data from the user. It contains the `properies` option to add variables to the object as follows:
+
+```yaml
+  parameters:
+    properties:
+      person:
+        type: object
+        properties:
+          name:
+            type: string
+          age:
+            type: number
+```
+
+You may choose to make an object property to be mandatory using the `required` property.
+
+```yaml
+  parameters:
+    properties:
+      person:
+        type: object
+        required:
+          - name
+        properties:
+          name:
+            type: string
+          age:
+            type: number
+```
+
+## `array`
+
+You can prompt for an array of properties using the array option. The `items` option can be any type: `array`, `object`, `string` or `number` as you like.
+
+```yaml
+  parameters:
+    properties:
+      languages:
+        type: array
+        items:
+          type: string
+```
+
+If you would like to prompt the user to add entity tags, you can use the `ui:field: EntityTagPicker` as shown below.
+
+```yaml
+  parameters:
+    properties:
+      entityTags:
+        type: array
+        ui:field: EntityTagsPicker
+```
+
+## Common Options
+
+If you would like to default the value of a field you can use the `default` option:
+
+```yaml
+  parameters:
+    properties:
+      name:
+        type: string
+        default: "world!"
+```
+
+If you would like to prompt the users for a fixed list of options, you may use the `enum` option.
+
+```yaml
+  parameters:
+    properties:
+      size:
+        type: number
+        enum: [50, 100, 200]
+```
+
+You can display a more human description to a field value by using `title` and `description`
+
+```yaml
+  parameters:
+    properties:
+      name:
+        type: string
+        title: "Name"
+        description: "Name to say hello to"
+```
+
+## Form Steps
+It might be jarring for your user to enter a lot of parameters one after another on the same page, especially if some of the propertiers require validation. As such Backstage have provided form steps.
+
+You can make use of form steps using the following example.
+
+```yaml
+  parameters:
+    - title: "Fill in the Name"
+      properties:
+        name:
+          type: string
+    - title: "Fill in the Age"
+      properties:
+        age:
+          type: number
+```
+
+## More Reading
+
+You can read more about parameter configuration in the official backstage docs here https://backstage.io/docs/features/software-templates/writing-templates.
```

Comment:
This functionality is provided to Backstage via another library right? Does it make sense to link to the docs of that library also?

Reply:
good idea.

Reply:
I looked into this a little. There is a sort of partial support for the uiSchema, and its behaviour only partially works. The linked page already links and references nuances about the uiSchema lib. So I think that is enough, what do you think?